packages/theme/src/client/module/blog/styles/tag-list.scss

```diff
@@ -1,63 +1,15 @@
 @use "sass:color";
 @use "sass:list";
 
-.tag-list-wrapper {
-  position: relative;
-  z-index: 2;
-  padding-left: 0;
-  list-style: none;
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: flex-start;
+.tag-list.tags-wrapper {
+  margin: 4vh 0;
```

Comment:
Why `vh` here, I am trying to make theme compatble when user are using different root font-size. I have friend who is  visually impaired so I care about a11y, all styles should be stick to `rem` if possible, then `px`.

Reply:
I used vh because it's I wanted a gap around the tag list that was related to the screen size, and not to the base font size. I also have minor sight problems, and often zoom up a web page. I personally hate it when gaps that were inserted to space things out vertically increase at the same time, because it means that text moves off the bottom of the screen.

I guess it's a difference between 1rem meaning "base font size" and 1rem meaning "current zoom factor" (ish).

I can easily change it if you really want. I just wanted to give you my reasoning.